### ISSUE TITLE
Cannot use webpack chunks name comment

### ISSUE THREAD

**Issue by PengWang0316** [theKashey/react-imported-component#16]
Hi there,

This issue has hunt me for a long time. Please give me a little bit help.
When I try to use comments in the code to create a webpack chunks name like:
```
const Login = importedComponent(() => import(/* webpackChunkName: 'LoginContainer' */ './containers/LoginContainer'), { LoadingComponent: LoadingAnimation });
```
I always get a number for the bundle name instead of LoginContainer. The result as below:
![image](https://user-images.githubusercontent.com/24322974/40078755-6bf6d28a-583a-11e8-8780-cc20dab75360.png)
If I use system.import instead of import, it will work just fine. All webpack chunk name can be displayed appropriately. The code as below:
```
const Login = importedComponent(() => system.import(/* webpackChunkName: 'LoginContainer' */ './containers/LoginContainer'), { LoadingComponent: LoadingAnimation });
```

Here is part of my webpack config: (version is 4.8.3)
```
output: {
    path: path.resolve(__dirname, 'dist'),
    filename: '[name].bundle.js',
    chunkFilename: '[name].bundle.js',
    publicPath: '/'
  },
```
Could someone please give me a little bit hint about where I did wrong? Really appreciate.

**Comment by theKashey** [comment]
Probably this information could be lost during babel transformation - it does not like comments :(

**Comment by PengWang0316** [comment]
Thank you for your quick response. I do not think it is babel's problem. Because it will work just fine, if I use system.import instead of import.

**Comment by theKashey** [comment]
You were right. Comments still exists after the transformation, but I might know where the problem lies. Will fix asap.

**Comment by theKashey** [comment]
v4.4.0.

The problem was with "generate-imported-component" script, which strips webpack comments out of import, meanwhile ImportedComponent uses that file to perform initial load, and webpack also sees it, and generates name-less chunks.

**Comment by PengWang0316** [comment]
Thank you for the quick fix. However, I still get number for the chunk name after I updated to v4.4.0
Is there any special Webpack setting up I have to do? Should I reopen an issue?
 
![image](https://user-images.githubusercontent.com/24322974/40140703-689fa6d8-5908-11e8-8f7c-d9eca79d1769.png)

**Comment by theKashey** [comment]
Just to double check - could you try /* webpackChunkName: "LoginContainer" */ (different braces)

**Comment by theKashey** [comment]
And dont forget to `regenerate` imported file, and double check that chunks name are ok
(i mean file like this - https://github.com/theKashey/react-imported-component/blob/master/examples/SSR/typescript-react/app/imported.ts )

**Comment by PengWang0316** [comment]
I tried to use ', " ,and without both of them. Also tried chunks name as your example. But it still not work for me. What do you mean about regenerate imported file?

**Comment by theKashey** [comment]
> What do you mean about regenerate imported file?
Look like you are not using SSR, so forget about it. But that also mean that I did fix a bug, but that was not bug of yours...

**Comment by PengWang0316** [comment]
No, I am not using SSR. Really appreciate you spending time on this library. I hope you can also find this bug. 😸

**Comment by PengWang0316** [comment]
After I removed dynamic-import-webpack from babelrc, everything works fine. Thank you for your help. Really appreciate.
